Data/ByteString/Lazy.hs

```diff
@@ -41,6 +41,11 @@
 --
 -- > import qualified Data.ByteString.Lazy as B
 --
+-- Note: Using functions relying on lazy `hGetContents` implies that you
```

Comment:
It's purely stylistic, but I would rather maintain existing convention of `bytestring` haddocks and stick to single quotes instead of backticks.